nailgun/entities.py

```diff
@@ -1670,12 +1670,21 @@ def create_missing(self):
                 'Found instance attributes: {0}'.format(self.get_values())
             )
         super(Host, self).create_missing()
+        self.name = self.name.lower()
```

Comment:
Gotcha. Well, that's some annoying server-side behaviour.

/me checks for a BZ about that.